applications/nrf_desktop/src/modules/dvfs.c

```diff
@@ -35,32 +35,50 @@ enum state {
 
 static enum state module_state = STATE_DISABLED;
 
+#define FOR_DVFS_STATE_LIST(MACRO, ...)				\
```

Comment:
Verb at the beginning to align the naming with other macros in this module

```suggestion
#define LIST_DVFS_STATE(MACRO, ...)				\
```